core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLBatchTestBase.java

```diff
@@ -110,7 +113,9 @@ public static void setupTest() throws Exception {
     addPluginArtifact(Id.Artifact.from(Id.Namespace.DEFAULT, "transforms", "1.0.0"), APP_ARTIFACT_ID,
                       ProjectionTransform.class, ScriptFilterTransform.class,
                       ValidatorTransform.class, CoreValidator.class,
-                      StructuredRecordToGenericRecordTransform.class);
+                      StructuredRecordToGenericRecordTransform.class,
+                      JavaScriptTransform.class,
+                      PythonEvaluator.class, PythonInterpreter.class);
```

Comment:
hm good point. Not sure why I added this, ran tests without it and they all pass. Will remove.